node/src/BaseClient.ts

```diff
@@ -266,6 +269,11 @@ function getRequestErrorClass(
     return RequestError;
 }
 
+export type PubSubMsg = {
+    message: string;
```

Comment:
does string support BulkString (non-UTF8) ?

Reply:
no, still waiting to see how the design for bytes will be in node

Reply:
ack